internal/app/api/api.go

```diff
@@ -74,6 +74,31 @@ func Init(conns *InfraConns) (http.Handler, error) {
 			method:  get,
 			handler: indexWebHandler.Index,
 		},
+		{
+			path:    "/customer",
+			method:  "POST",
+			handler: indexWebHandler.AddFriend,
+		},
+		{
+			path:    "/customer/{id}",
+			method:  "GET",
+			handler: indexWebHandler.FindFriendByID,
+		},
+		{
+			path:    "/customer",
```

Comment:
should be:
GET /customers --> without body
DELETE, GET /customers/{id} --> without body
PUT /customers/{id} --> with body

Reply:
Updated